include/clad/Differentiator/ReverseModeVisitor.h

```diff
@@ -412,6 +412,7 @@ namespace clad {
     StmtDiff VisitCaseStmt(const clang::CaseStmt* CS);
     StmtDiff VisitDefaultStmt(const clang::DefaultStmt* DS);
     DeclDiff<clang::VarDecl> DifferentiateVarDecl(const clang::VarDecl* VD);
+    DeclDiff<clang::VarDecl> DifferentiateVarDecl(const clang::VarDecl* VD, bool AddToBlock);
```

Comment:
```suggestion
    DeclDiff<clang::VarDecl> DifferentiateVarDecl(const clang::VarDecl* VD, bool AddToBlock = true);
```

That may simplify your life and you won't need to define another overload below.